examples/demos/dnd/dropActions.js

```diff
@@ -0,0 +1,23 @@
+import moment from 'moment'
+
+export function updateEventTime (event, newDate) {
+  const { start, end } = event
+  const { value, type } = newDate
+  const dayDifference = moment(end).diff(moment(start), 'days')
+  const timeDifference = moment(end).diff(moment(start), 'hours')
+
+// Time slots for week/day views
+  const newStartTimeSlot = `${moment(value).format('HH:mm:sss')}`
+  const newEndTimeSlot = `${moment(value).add(timeDifference, 'hours').format('HH:mm:sss')}`
+
+  const newStartDate = `${moment(value).format('MM/DD/YYYY')} ${type === 'Day' ? moment(start).format('HH:mm:ss') : newStartTimeSlot}`
+  const newEndDate = `${moment(value).add(dayDifference, 'days').format('MM/DD/YYYY')} ${type === 'Day' ? moment(end).format('HH:mm:ss') : newEndTimeSlot}`
+
+  const newEventTimes = {
+    ...event,
+    start: moment(newStartDate).toDate(),
+    end: moment(newEndDate).toDate()
+  }
+
+  return newEventTimes
```

Comment:
Would it be possible to perform the `start` and `end` calculations directly through the `moment`s, without converting to strings and parsing them?